snuba/subscriptions/entity_subscriptions/generic_metrics/distributions.yaml

```diff
@@ -0,0 +1,7 @@
+version: v1
+kind: entity_subscription
+name: generic_metrics_distributions_subscription
+
+parent_subscription_entity_class: sessions_subscription
```

Comment:
I'm pretty against specifying parent classes here -- I'd much rather duplicate information than have this thing support inheritance

Reply:
Changed PluggableEntitySubscription to only extend from SessionSubscription as mentioned in comment below. We can revisit this once we migrate datasets that require a different base class.